File: C/ex05.c
//5. Entrar via teclado com o valor de uma temperatura em graus Celsius, calcular 
//e exibir sua temperatura equivalente em Fahrenheit.

#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
    // Declaração de variáveis
  float gc, f;
  // Solicita entrada de dados para o usuário e grava o valor digitado em uma variável "gc"
  printf("Digite a temperatura em Graus Celsius: ");
  scanf("%f", &gc);

  //faz o calculo de celsius para fahrenheit
 f=(gc*1.8)+32;
 // Exibe o resultado final
  printf("A temperatura %.2f em celsius equivale a %.2f em fahrenheit",gc, f); 


    return 0;
}

File: Python/ex05.py
# Entrar via teclado com o valor de uma temperatura em graus Celsius, calcular e exibir sua 
# temperatura equivalente em Fahrenheit.

temp = float(input("Digite o valor da temperatura em graus Celsius: "))
 
convers = (temp*1.8+32)
 
print(f"O valor de {temp:.1f} graus Celsius em Fahrenheit é: {convers:.1f}")

